### PR TITLE
Fix missing tilde

### PR DESCRIPTION
**Pull request by benklop**
Fix broken Tilde and Grave (Backtick/Backquote)

This was copied from https://github.com/debauchee/barrier/pull/1711 by
@dgentry.

Resolves #1541 and #1407.

## Contributor Checklist:

* [x] I have created a file in the `doc/newsfragments` directory *IF* it is a
      user-visible change (and make sure to read the `README.md` in that directory)

### CODE REVIEW

**Comment by benklop** [comment]
I decided to open this from https://github.com/debauchee/barrier/pull/1711 since it didn't seem like anyone else was doing it. Seems like a simple enough fix. I'm not familiar enough with JIS layouts to know if this might cause some other issue, but it certainly is better (for me) than having non-working tilde and backticks.

**Comment by shymega** [comment]
A couple of feedback:

- Can you adjust the PR description and fix the commit to add the 'Fixes' part from your newsfragment.
- Finally, squashing both your commits into one would be preferable. Whilst I need a review from @p12tic, I can't merge this commit as-is.

**Comment by shymega** [comment]
Alternatively, I can rebase and edit the commits myself, and force-push to the PR - we have access to do so. Would that be preferable?

**Comment by shymega** [comment]
I didn't hear back, and because we have a lot of PRs left, I decided to rebase, squash the two commits, amend the main commit, and rebase against mainline.

Once checks pass, we can look at merging.

**Review by p12tic — APPROVED** [review]

**Comment by dgentry** [comment]
This is great, thank you.

**Comment by benklop** [comment]
hey, sorry for leaving this hanging. I appreciate you proceeding!

**Pull request merged**
